modules/installation-configuration-parameters.adoc

```diff
@@ -1671,7 +1671,7 @@ Additional Azure configuration parameters are described in the following table:
 |String
 
 |`platform.azure.resourceGroupName`
-| The name of an already existing resource group to install your cluster to. If undefined, a new resource group is created for the cluster.
+|The name of an already existing resource group to install your cluster to. This resource group must be empty and only used for this specific cluster; the cluster components assume ownership of all resources in the resource group. If you limit the service principal scope of the installation program to this resource group, you must ensure all other resources used by the installation program in your environment have the necessary permissions, such as the public DNS zone and virtual network. Destroying the cluster using the installation program deletes this resource group.
```

Comment:
```suggestion
|The name of an already existing resource group to install your cluster to. This resource group must be empty and only used for this specific cluster; the cluster components assume ownership of all resources in the resource group. If you limit the service principal scope of the installation program to this resource group, you must ensure all other resources used by the installation program in your environment have the necessary permissions, such as the public DNS zone and virtual network. Destroying the cluster by using the installation program deletes this resource group.
```
https://www.ibm.com/docs/en/ibm-style?topic=word-usage

Reply:
Hi @obrown1205, will you add a preview link for verifying the output and also specify what version of the docs this update is for (i.e. 4.11+, 4.12, etc)?  Thanks!